main/res/values/preference_keys.xml

```diff
@@ -307,4 +306,17 @@
     <string translatable="false" name="pref_compacticon_auto">auto</string>
 
     <string translatable="false" name="pref_selected_language">selectedLanguage</string>
+
+    <!-- theme preferences -->
+    <string translatable="false" name="pref_skin_depreciated">skin</string>
```

Comment:
Can you rename this preference to `old_pref_skin`, please, to follow name conventions for deprecated preference keys? Thanks.

Reply:
Done